main.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+from WonderPosition import *
+from WonderWoerter import *
+from WonderBlumen import *
+from WonderUmgebung import *
+from WonderBewegung import *
+from WonderTexte import *
+from WonderInventar import *
+
+
+class Spiel(object):
+    def __init__(self):
+        self.wortVergleicher = WortVergleicher()
+        self.wondertext = WonderText()
+        self.wonderInventar = WonderInventar(self.wortVergleicher)
+        self.umgebungen = UmgebungsGenerator(self.wortVergleicher)      
+        self.bewegung = Bewegung(self.umgebungen.gebeStartUmgebung(),self.wondertext, self.wortVergleicher)
+        self.umgebungen.gebeStartUmgebung().setzeBewegung(self.bewegung)
+        self.DoktormutterPosition = self.umgebungen.gebeDoktormutterPosition()
+        self.umgebung =  self.umgebungen.gebeStartUmgebung()
+
+    def spiele(self):
+
+        self.wondertext.setzeText("=================WonderDoktorandin========================" + "\n")
+        self.wondertext.ergaenzeText("Du bist eine Doktorandin der Biologie. Deine Aufgabe ist es, drei rießige Blumen zu finden und dann zu Deiner Doktormutter zu bringen." + "\n") 
+        self.wondertext.ergaenzeText("Deine Doktormutter befindet sich an der Position (X.Y.Z=)" + str(self.DoktormutterPosition.gebeX()) +  "." + str(self.DoktormutterPosition.gebeY()) + "." + str(self.DoktormutterPosition.gebeZ()) )   
```

Comment:
Der idiomatische Weg in python hierfür ist
```suggestion
        self.wondertext.ergaenzeText(f"Deine Doktormutter befindet sich an der Position (X.Y.Z=) {str(self.DoktormutterPosition.gebeX())}.{str(self.DoktormutterPosition.gebeY())}.{str(self.DoktormutterPosition.gebeZ())}")  
```